### PR TITLE
Add task model

### PR DESCRIPTION
**Pull request by leowangx2013**
Commits follow the M, V, C order.

### CODE REVIEW

File: app/views/layouts/missionsystem.html.slim [hunk 1]
```diff
@@ -0,0 +1,24 @@
+doctype html
+html
+  head
+    title
+      | Ruby on Rails Tutorial Sample App |
```
Comment (xzhflying) [inline app/views/layouts/missionsystem.html.slim]: .......

File: app/views/layouts/missionsystem.html.slim [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      = yield(:title)
+    = stylesheet_link_tag   "application", media: "all",
+                                              "data-turbolinks-track" => true
+    = javascript_include_tag "application", "data-turbolinks-track" => true
+    = csrf_meta_tags
+  body
+    header class="navbar navbar-fixed-top navbar-inverse"
+      div class="navbar-inner"
+        div clas="container"
+          = link_to "sample app", "#", id: "logo"
```
Comment (xzhflying) [inline app/views/layouts/missionsystem.html.slim]: why called sample app?

File: app/views/layouts/missionsystem.html.slim [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          nav
+            ul class="nav pull-right"
+              li = link_to "Home",    missionsystem_main_path
+              li = link_to "Help",    missionsystem_help_path
+              li = link_to "About",   missionsystem_about_path
+
```
Comment (xzhflying) [inline app/views/layouts/missionsystem.html.slim]: Try to use I18n on these things

File: app/views/layouts/missionsystem.html.slim [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    div class="container"
+      | asdfsadfsdf
```
Comment (xzhflying) [inline app/views/layouts/missionsystem.html.slim]: hmm.... If you mean `test`, please use `test` or `TBD`...

File: app/views/static_pages/missionsystem_about.html.slim [hunk 5]
```diff
@@ -0,0 +1,10 @@
+= provide(:title, 'About')
+div class="center hero-unit"
+  h1 About Us
```
Comment (xzhflying) [inline app/views/static_pages/missionsystem_about.html.slim]: Also you should consider I18n, but you can do it in the future.

File: app/views/static_pages/missionsystem_about.html.slim [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+  h2
+    | This is the about page for the
+    a<> href="http://railstutorial.org/"
+      Ruby on Rails Tutorial
+    sample application.
+
```
Comment (xzhflying) [inline app/views/static_pages/missionsystem_about.html.slim]: this part seems to be not correct.

File: app/views/static_pages/missionsystem_help.html.slim [hunk 7]
```diff
@@ -0,0 +1,10 @@
+= provide(:title, 'Help')
+div class="center hero-unit"
+  h1 Help
+  h2
+    | This is the help page for the
+    a<> href="http://railstutorial.org/"
+      Ruby on Rails Tutorial
+    sample application.
```
Comment (xzhflying) [inline app/views/static_pages/missionsystem_help.html.slim]: this part seems to be not correct.

File: app/views/static_pages/missionsystem_main.html.slim [hunk 8]
```diff
@@ -0,0 +1,10 @@
+= provide(:title, 'Home')
+div class="center hero-unit"
+  h1 Welcome to the Sample App
+  h2
+    | This is the home page for the
+    a<> href="http://railstutorial.org/"
+      Ruby on Rails Tutorial
+    sample application.
```
Comment (xzhflying) [inline app/views/static_pages/missionsystem_main.html.slim]: this part seems to be not correct.

File: db/migrate/20150921134132_add_supe_to_tasks.rb [hunk 9]
```diff
@@ -0,0 +1,5 @@
+class AddSupeToTasks < ActiveRecord::Migration
+  def change
+    add_column :tasks, :supe, :string
```
Comment (xzhflying) [inline db/migrate/20150921134132_add_supe_to_tasks.rb]: hmm... what is supe?

**Comment by xzhflying** [comment]
And where is your complete migration?

File: db/migrate/20150921134132_add_supe_to_tasks.rb (hunk 9 above)
Comment (leowangx2013) [inline db/migrate/20150921134132_add_supe_to_tasks.rb]: supe means "supervisor"

Comment (xzhflying) [inline db/migrate/20150921134132_add_supe_to_tasks.rb]: Do not use abbreviation in the schema...

File: app/views/layouts/missionsystem.html.slim (hunk 1 above)
Comment (leowangx2013) [inline app/views/layouts/missionsystem.html.slim]: The code is copied from the Rails Tutorial. Later I'll write my own page content

File: app/views/layouts/missionsystem.html.slim (hunk 2 above)
Comment (leowangx2013) [inline app/views/layouts/missionsystem.html.slim]: This is still from the rails tutorial, I'll change it later

File: app/views/layouts/missionsystem.html.slim (hunk 3 above)
Comment (leowangx2013) [inline app/views/layouts/missionsystem.html.slim]: ok

File: app/views/layouts/missionsystem.html.slim (hunk 4 above)
Comment (leowangx2013) [inline app/views/layouts/missionsystem.html.slim]: ok

File: app/views/static_pages/missionsystem_about.html.slim (hunk 6 above)
Comment (leowangx2013) [inline app/views/static_pages/missionsystem_about.html.slim]: I see. I'll change it.

File: db/migrate/20150921134132_add_supe_to_tasks.rb (hunk 9 above)
Comment (leowangx2013) [inline db/migrate/20150921134132_add_supe_to_tasks.rb]: It's a new column that added to the Task table

**Pull request closed without merging**